index.js

```diff
@@ -1,17 +1,53 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
 
-  get(pos) {}
+  }
 
-  max() {}
+  get(pos) {
+    if (!this.items[pos]) {
+      throw new Error('OutOfBounds');
+    }
+    return this.items[pos];
+  }
 
-  min() {}
+  max() {
+    if (!this.length) {
+      throw new Error('EmptySortedList');
+    }
+    return this.items[this.length - 1];
+  }
 
-  sum() {}
+  min() {
+    if (!this.length) {
+      throw new Error('EmptySortedList');
+    }
+    return this.items[0];
+  }
 
-  avg() {}
+  sum() {
+    if (!this.length) {
+      return this.length;
+    } else {
```

Comment:
devolver solo esto `return this.items.reduce((acc, item) => acc + item, 0)` daria el mismo resultado